chart/tests/ingress_test.yaml

```diff
@@ -312,3 +312,15 @@ tests:
           nginx.ingress.kubernetes.io/proxy-connect-timeout: "30"
           nginx.ingress.kubernetes.io/proxy-read-timeout: "1800"
           nginx.ingress.kubernetes.io/proxy-send-timeout: "1800"
+- it: it should fail when disableHTTP is set and servicePort is not 443
+  set:
+    hostname: host.example.com
+    ingress:
+      enabled: false
+    httpRoute:
+      enabled: true
```

Comment:
```suggestion
```

Based on the name of this test case I think we only need to include the `service` setting.